### DISCUSSION TITLE
有谁知道发布的贴子怎么删除啊

### DISCUSSION THREAD

**Discussion by L-walnut** [share121/inter-knot#3382] [其他]
![屏幕截图 2024-08-20 200144](https://github.com/user-attachments/assets/bafc3311-e8d0-42c7-85d0-d9e7f09501d7)
有谁知道发布的贴子怎么删除啊？

**Comment by jiangshixie** [comment]
先点击自己发布的帖子，然后旁边会有一个垃圾桶贴图，点击就删除了<img src="https://bbs-static.miyoushe.com/static/2024/06/19/52971140330611d82cad599f20788ab6_3848320906174398412.png" alt="帕姆 开拓" width="75" >

**Comment by hexu165** [comment]
涨知识了
